src/Microsoft.Net.Http.Headers/MultipartMediaTypeHeaderValueExtensions.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+
+namespace Microsoft.Net.Http.Headers
+{
+    /// <summary>
+    /// Various extension methods of <see cref="MediaTypeHeaderValue"/>
+    /// </summary>
+    public static class MultipartMediaTypeHeaderValueExtensions
+    {
+        /// <summary>
+        /// Check if the media type field if application form
+        /// </summary>
+        /// <param name="contentType">The header value to check</param>
+        /// <returns>True if field is application form, false otherwise</returns>
+        public static bool IsApplicationFormContentType(this MediaTypeHeaderValue contentType)
```

Comment:
Do these need to be on `MediaTypeHeaderValue`? Can't some of these hang off HttpRequest?